### PR TITLE
CMS-354 migrate settings setup

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-354
- please ignore compiled css file
- and please ignore editorconfig file as well

### CODE REVIEW

File: djangocms_admin_style/sass/settings/_all.scss [hunk 1]
```diff
@@ -1,2 +1,2 @@
 @import "cms.scss";
```
Comment (vxsx) [inline djangocms_admin_style/sass/settings/_all.scss]: should be just `cms` no need for extension

File: djangocms_admin_style/sass/settings/_all.scss [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
-@import "_custom.scss";
\ No newline at end of file
+@import "_custom.scss";
```
Comment (vxsx) [inline djangocms_admin_style/sass/settings/_all.scss]: no need for extension and no need for explicit `_` here

File: djangocms_admin_style/sass/djangocms-admin.scss [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-@import "partials/_base.scss";
-@import "partials/_forms.scss";
-@import "partials/_tables.scss";
-@import "partials/_filer.scss";
-@import "partials/_changelist.scss";
-@import "partials/_cmsplaceholders.scss";
-@import "partials/_content.scss";
-@import "partials/_icons.scss";
-@import "partials/_footer.scss";
+@import "mixins/_all.scss";
```
Comment (vxsx) [inline djangocms_admin_style/sass/djangocms-admin.scss]: all the underscores and file extensions are not necessary

**Comment by vxsx** [comment]
LGTM :+1:

**Comment by Brankub** [comment]
:+1:

**Pull request merged**